solutions/quickstart/README.md

```diff
@@ -49,4 +49,4 @@ To view your current configuration, run the `terraform refresh` command.
 1. Export API Key
     export TF_VAR_ibmcloud_api_key=xxxx
 2. Run Terraform init
-3. Run Terraform apply
+3. Run Terraform apply
```

Comment:
You didn't make a change to the README but you must have changed something for it to show up in the PR.  Please checkout the original README again so it doesn't show under the PR.